Link to "Party Cluster" dead, replaced with valid one

oliviervaillancourt opened the pull request:
Change the link of party cluster from `http://tryazureservicefabric.westus.cloudapp.azure.com/` (resulting in *DNS error*) to `https://try.servicefabric.azure.com/`

azurecla commented:
Hi __@cvrajeesh__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

oliviervaillancourt committed:
Link to "Party Cluster" dead, replaced with valid one

Change the link of party cluster from http://tryazureservicefabric.westus.cloudapp.azure.com/ (resulting in DNS error) to https://try.servicefabric.azure.com/
